openssh-server-pam.yaml

```diff
@@ -0,0 +1,84 @@
+package:
+  name: openssh-server-pam
+  version: "9.9_p2"
+  epoch: 0
+  description: "the OpenBSD SSH implementation"
+  copyright:
+    - license: ISC
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libedit-dev
+      - linux-pam-dev
+      - openssl-dev
+      - wolfi-baselayout
+      - zlib-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \_(p\d+)$
+    replace: $1
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-${{vars.mangled-package-version}}.tar.gz
+      expected-sha256: 91aadb603e08cc285eddf965e1199d02585fa94d994d6cae5b41e1721e215673
+
+  - name: Configure
+    runs: |
+      ./configure \
+         --host=${{host.triplet.gnu}} \
+         --target=${{host.triplet.gnu}} \
+         --prefix=/usr \
+         --datadir=/usr/share \
+         --sysconfdir=/etc/ssh \
+         --libexecdir=/usr/lib/ssh \
+         --with-pid-dir=/run \
+         --with-mantype=doc \
+         --disable-utmp \
+         --disable-wtmp \
+         --disable-lastlog \
+         --disable-strip \
+         --with-privsep-path=/var/empty \
+         --with-xauth=/usr/bin/xauth \
+         --with-default-path='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin' \
+         --with-privsep-user=sshd \
+         --with-ssl-engine \
+         --with-libedit \
+         --with-pam --with-pam-service=sshd
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+  - name: "openssh-server-pam"
+    runs: |
+      mkdir -p "${{targets.contextdir}}"/usr/lib/ssh
+      mkdir -p "${{targets.contextdir}}"/etc/pam.d
+      mkdir -p "${{targets.contextdir}}"/usr/sbin
+      install -m755 sshd ${{targets.contextdir}}/usr/sbin/sshd.pam
+      install -m755 sshd-session ${{targets.contextdir}}/usr/lib/ssh/sshd-session
+      install -Dm644 ${{targets.contextdir}}/usr/sbin/sshd.pam ${{targets.contextdir}}/etc/pam.d/sshd
```

Comment:
These lines would go in `openssh.yaml` in a subpackage called `openssh-pam`, similar to the `openssh-sftp-server` subpackage.